<?xml version="1.0" encoding="UTF-8"?>

<launch>
  <node name="turtlesim" pkg="turtlesim" type="turtlesim_node">
    <remap from="turtle1/cmd_vel" to="turtle1/cmd_vel_reversed" />
  </node>

  <node name="teleop_key" pkg="turtlesim" type="turtle_teleop_key" required="true" launch-prefix="xterm -e"/>

  <node name="reverse_velocity" pkg="jessica" type="reverse_cmd_velocity_node" output="screen"/>
</launch>
